docs/testplan/Add-cluster-with-GCU.md

```diff
@@ -0,0 +1,127 @@
+- Verify the route table in the downstream DUT host to ensure that the static route is visible.
+
+Remove Peers from Downstream Namespace:
+- Remove all BGP neighbors for the selected ASIC namespace via apply-patch.
+- Verify the route table. All routes for BGP neighbors should be gone. Additionally, all kernel and directly connected routes toward the neighbor IPs should be removed.
+- Shutdown local interfaces for the selected ASIC namespace via apply-patch.
+- Verify that the admin status of local interfaces in the selected ASIC namespace is down/down.
+- Perform data traffic tests toward a randomly selected neighbor. Traffic should fail.
+- Perform data traffic tests toward the static route from the randomly selected neighbor. Traffic should fail.
+
+Re-add Peers and Re-enable Interfaces:
+- Change cable lengths.
+- Re-add peers in the downstream namespace. -
+- Re-enable interfaces.
+- Verify that the peers are re-added, BGP sessions are established, and the route table is updated.
+- Verify the buffer profile exists in CONFIG_DB, APPL_DB, and ASIC_DB.
+
+#### Testing Steps
+
+Data traffic verifications:
+- Perform data traffic tests toward a randomly selected neighbor/the static route from the randomly selected neighbor. Traffic should pass. Verify there are no packet drops via checking pkt counters.
+- Perform data traffic from upsteram to downstream linecard (interlinecard). Perform data traffic from the other asic of the same downsteram linecard (innerlinecard).
+- Variation with acl config (DATAACL, CTRLPLANE) verifies traffic passes/drops based on acl rules and src/dst port criterion.
+
+[to be enhanced]
+As a test variation, instead of applying changes per ASIC namespace, the same scenario shall be adapted to apply changes to a randomly selected interface that is a member of a portchannel from a randomly selected namespace.
+
+### Test Case # 2 - Update CABLE Length
+
+#### Test Objective
+To verify updates in config path "CABLE_LENGTH".
+
+#### Testing Steps
+
+- Select a random ASIC namespace and shut down the interfaces.
+- Update cable length via apply-patch. Identify the current cable length and add the previous or next supported length value for this frontend card.
+- Bring the interfaces back up via apply-patch.
+- Verify that the interfaces are up.
+- Verify in CONFIG_DB and APPL_DB that the new cable length is applied.
+- Verify that updated pg lossless profile was created in CONFIG_DB and APPL_DB and that it was assigned to active interfaces.
+
+
+### Test Case # 3 - Load QoS
+
+#### Test Objective
+To verify qos updates in multi-asic t2 platform. To verify updates in tables "BUFFER_PG", "BUFFER_QUEUE", "PORT_QOS_MAP", and "QUEUE".
+
+#### Testing Steps
+
+- Select a random ASIC namespace and shut down the interfaces. (to be modified per interface)
+- Remove QoS config via apply-patch remove operation for tables "BUFFER_PG", "BUFFER_QUEUE", "PORT_QOS_MAP", and "QUEUE".
```

Comment:
I find the QUEUE and BUFFER_QUEUE tables are populated already irrespective of whether the interface is present/enabled 0--- as it is based on the system ports, which is predefined.